cypress/e2e/tests/pages/explorer/workloads/daemonsets.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { WorkloadsDaemonsetsListPagePo, WorkLoadsDaemonsetsEditPagePo } from '@/cypress/e2e/po/pages/explorer/workloads-daemonsets.po';
+import { generateFakeClusterDataAndIntercepts } from '@/cypress/e2e/blueprints/nav/fake-cluster';
+const fakeProvClusterId = 'some-fake-cluster-id';
+const fakeMgmtClusterId = 'some-fake-mgmt-id';
+
+describe('Cluster Explorer', { tags: ['@explorer', '@adminUser'] }, () => {
+  beforeEach(() => {
+    generateFakeClusterDataAndIntercepts(fakeProvClusterId, fakeMgmtClusterId);
+
+    // to test payload of https://github.com/rancher/dashboard/issues/9874
+    cy.intercept('PUT', `k8s/clusters/${ fakeMgmtClusterId }/v1/apps.daemonsets/calico-system/calico-node`, (req: any) => {
+      req.reply({
+        statusCode: 200,
+        body:       {}
+      });
+    }).as('daemonsetEdit');
+
+    // to prevent error when rendering the daemonsets list after save (otherwise test will fail)
+    cy.intercept('GET', `k8s/clusters/${ fakeMgmtClusterId }/v1/nodes`, (req: any) => {
+      req.reply({
+        statusCode: 200,
+        body:       { data: [] }
+      });
+    }).as('nodesReq');
+
+    cy.login();
+  });
+
+  describe('Workloads', () => {
+    describe('Daemonsets', () => {
+      it('modifying "Scaling and Upgrade Policy" to "On Delete" should use the correct property "OnDelete"', () => {
+        // same name as in blueprints/nav/k8s-schemas/apps.daemonsets
+        const daemonsetName = 'calico-node';
+
+        // list view daemonsets
+        const workloadsDaemonsetsListPage = new WorkloadsDaemonsetsListPagePo(fakeMgmtClusterId);
+
+        workloadsDaemonsetsListPage.goTo();
+        workloadsDaemonsetsListPage.listElementWithName(daemonsetName).should('exist');
```

Comment:
I think we would prefer asserting the visibility instead

Reply:
change done. I verified that works, but fundamentally `exists` is different than `visible`... Imagine that the daemonset row exists but is not visible in the table (needs scrolling). That doesn't mean it doesn't exist on the list... It should be clickable by the user. 

The `visible` assertion would fail on that particular case.

Reply:
@aalves08 that's a very good point, my concern with `exists` is that even a hidden element with zero width and height can still pass the test, but as you mentioned there are some edge cases as well.